mftf/2.3/tips-tricks.md

```diff
@@ -6,26 +6,82 @@ Sometimes, little changes can make a big difference in your project. Here are so
 
 <!-- {% raw %} -->
 
-## Use descriptive variable names
+## Actions and action groups
 
-Use descriptive variable names to increase readability.
-It makes the code easier to follow and update.
+### NEW! Perform the most critical actions first in the `<after>` block
 
- {:style="color:green"}
-GOOD:
+Perform actions such as `magentoCLI` and `deleteData` first because these are more likely to succeed since no UI is involved.
+In the bad example, we perform some heavy UI steps first.
```

Comment:
This line threw me off a bit since the bad example had not been introduced yet and there was no indication on where it to find this example. Maybe move this to after the examples are presented?

Reply:
Nice. Fixed up.